resume-buddy/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -33,14 +34,17 @@
 @WebServlet("/login")
 public class LoginServlet extends HttpServlet {
 
+  public static String userType;
```

Comment:
This should be a private, non-static field.

```
private UserType userType;
```

You can expose the value of it with a getter:
```
public UserType getUserType() { return userType; }
```